domains/yashk.json

```diff
@@ -0,0 +1,18 @@
+{
+  "subdomain": "yashk",
+  "owner": {
+    "email": "yashkulkarni2008@gmail.com",
+    "github": "0day-Yash"
+  },  "record": {
+    "MX": [
+      { "value": "mx.zoho.com", "priority": 10 },
+      { "value": "mx2.zoho.com", "priority": 20 },
+      { "value": "mx3.zoho.com", "priority": 50 }
+    ],
+    "TXT": [
+      { "name": "@", "value": "v=spf1 include:zoho.com ~all" },
+      { "name": "zoho-verification", "value": "zoho-verification-code" },
+    ]
```

Comment:
The hell? This is **NOT** how you do it. Please read our documentation on how to add MX and TXT records.